kolibri/plugins/learn/assets/src/views/exam-list/index.vue

```diff
@@ -1,10 +1,7 @@
 <template>
 
   <div>
-    <div v-if="!isUserLoggedIn" class="login-message">
-      <h1>{{ $tr('logInPrompt') }}</h1>
-      <p>{{ $tr('logInCommand') }}</p>
-    </div>
+    <auth-message v-if="!isUserLoggedIn" authorizedRole="learner" />
```

Comment:
There is a `isFacilityUser` getter in core getters that allows for this filtering.